javascript/javascript2/week3/Warmup.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        body {
+            height: 620px;
+        }
+    </style>
+
+<body>
+    <button id="but">Click me!</button>
+    <button id='location'>Log location</button>
```

Comment:
It is too small, but you can use the prettier extension in vs code, to have the same format for string ;)

Reply:
I do not underestand what you mean. Would you please let me know.